ui/src/api.ts

```diff
@@ -78,6 +78,57 @@ export async function deleteAnnotation(arxivId: string, id: number) {
   return await axios.delete(`/api/v0/papers/arxiv:${arxivId}/annotation/${id}`);
 }
 
+export async function addLibraryEntry(paperId: string, paperTitle: string) {
+  const s2ApiUrl = `https://www.semanticscholar.org/api/1/library/entries`;
```

Comment:
I wonder if we should have an S2 const urls file or obj of some sort. Might be helpful as we add more calls to have all the urls in a central place